src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py

```diff
@@ -0,0 +1,199 @@
+               help='Relative mount path for nfs. The nfs will be available at '
+                    '$AZ_LEARNING_MOUNT_ROOT/<relative_mount_path> folder.',
+               arg_group='File Server Mount')
+    c.argument('azure_file_share', options_list=('--afs-name',),
+               help='Name of the azure file share to mount. Please provide AZURE_BATCHAI_STORAGE_ACCOUNT and '
+                    'AZURE_BATCHAI_STORAGE_KEY environment variables containing storage account name and key.',
+               arg_group='Azure File Share Mount')
+    c.argument('afs_mount_path', options_list=('--afs-mount-path',),
+               help='Relative mount path for Azure File share. The file share will be available at '
+                    '$AZ_LEARNING_MOUNT_ROOT/<relative_mount_path> folder. If you to mount  more than one Azure '
+                    'Storage container, configure them in a configuration file and use --config option.',
+               arg_group='Azure File Share Mount')
+    c.argument('container_name', options_list=('--container-name',),
+               help='Name of Azure Storage container to mount. Please provide AZURE_BATCHAI_STORAGE_ACCOUNT and '
+                    'AZURE_BATCHAI_STORAGE_KEY environment variables containing storage account name and key. If you '
+                    'to mount more than one Azure Storage container, configure them in a configuration file and use '
+                    '--config option.',
+               arg_group='Azure Storage Container Mount')
+    c.argument('container_mount_path', options_list=('--container-mount-path',),
+               help='Relative mount path for Azure Storage container. The container will be available at '
+                    '$AZ_LEARNING_MOUNT_ROOT/<relative_mount_path> folder.',
+               arg_group='Azure Storage Container Mount')
+    c.argument('json_file', options_list=('--config', '-c'),
+               help='A path to a json file containing cluster create parameters '
+                    '(json representation of azure.mgmt.batchai.models.ClusterCreateParameters).',
+               arg_group='Advanced')
+
+with ParametersContext(command='batchai cluster resize') as c:
+    c.register_alias('resource_group', options_list=('--resource-group', '-g'), arg_type=resource_group_name_type)
+    c.register_alias('cluster_name', options_list=('--name', '-n'), help='Name of the cluster.')
+    c.argument('target', options_list=('--target', '-t'), help='Target number of compute nodes.')
+
+with ParametersContext(command='batchai cluster auto-scale') as c:
+    c.register_alias('resource_group', options_list=('--resource-group', '-g'), arg_type=resource_group_name_type)
+    c.register_alias('cluster_name', options_list=('--name', '-n'), help='Name of the cluster.')
+    c.argument('min', options_list=('--min',), help='Minimum number of nodes.')
+    c.argument('max', options_list=('--max',), help='Minimum number of nodes.')
```

Comment:
Typo: Minimum when I believe this should read maximum.

Reply:
done